source/common/buffer/buffer_impl.h

```diff
@@ -749,6 +753,30 @@ class OwnedImpl : public LibEventInstance {
   void commit(uint64_t length, absl::Span<RawSlice> slices,
               ReservationSlicesOwnerPtr slices_owner) override;
 
+  uint8_t* inlineReserve(uint64_t size) override {
+    // If no any slice in the buffer then create one new slice based on the expected continuous
```

Comment:
If there are no slices in the buffer, then ....

Reply:
get it.